ocs_ci/framework/conf/default_config.yaml

```diff
@@ -32,6 +32,7 @@ RUN:
   # This config file disables scale app pods to use OCS workers
   use_ocs_worker_for_scale: False
   load_status: None
+  save_logs: False
```

Comment:
@petr-balogh although this key (`save_logs`) is talking about logs, I think we still should consider it as a RUN parameter since it's not being passed as an env param, but rather defined in the test itself, and is being dynamically changed and tracked by a background thread. 

We can maybe change the param name, so it will reflect better the purpose of it. I don't have a better naming though.

Reply:
ok, but please please document this value anyway in README and write that his is not supposed to be changed by user but by the fixture itself.

Also I think you can rename to something like:
`save_live_pod_logs`

If it's not suppose to be changed by user I would even not put this value to default config and just make sure you are using .get mehod when you getting this value.

This case you don't have to maybe even document in readme as this value is not suppose to be changed by user at all.

Reply:
Thanks, added documentation.